### ISSUE TITLE
Optional fields are not blanked

### ISSUE THREAD

**Issue by 123dev** [magicdude4eva/graylog-jira-alarmcallback#2]
_Splitting from: [issue](https://github.com/magicdude4eva/graylog-jira-alarmcallback/issues/1)_

I noticed that if I leave **JIRA MD5 pattern(optional)** blank, then it saves as **[MESSAGE_REGEX]**, meaning that there is no way of turning off non-duplicate test
Although this does not apply to us, other users might want to allow duplicate tickets.

Same applies to **Message regex(optional)** if I leave it blank it gets saved as **([a-zA-Z_.]+(?!.*Exception): .+)**.

Thanks

**Comment by magicdude4eva** [comment]
Fixed in release [1.0.4](https://github.com/magicdude4eva/graylog-jira-alarmcallback/releases/tag/1.0.4)

**Comment by magicdude4eva** [comment]
I will leave this open as I want to see if I can not find a fix so that I can provide a default value and still store an empty field if the default is deleted. This seems to be a Graylog bug and is also in Graylog v2.

**Comment by 123dev** [comment]
Sounds good, thanks for being proactive. :)

**Comment by magicdude4eva** [comment]
Graylog currently does not seem to allow the deletion of the content of a optional configuration field if the field as a default value. The only way to make this work would mean to make it hacky - i.e. ask the user to enter `<empty>` in the field.

I have now logged [an issue](https://github.com/Graylog2/graylog2-server/issues/2108) with Graylog as I can not find a proper way to do this. For the time being, no optional field will have a pre-populated default value. Where possible, I have included the default value in the help-description.
